aspnetcore/signalr/version-differencs.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Differences between SignalR and ASP.NET Core SignalR
+author: rachelappel
+description: Differences between SignalR and ASP.NET Core SignalR 
+manager: wpickett
+monikerRange: '>= aspnetcore-2.1'
+ms.author: rachelap
+ms.date: 06/30/2018
+ms.prod: asp.net-core
+ms.technology: aspnet
+ms.topic: article
+uid: signalr/version-differences
+---
+
+# Differences between SignalR and ASP.NET Core SignalR
+
+ASP.NET Core SignalR is not compatible with clients or servers for ASP.NET SignalR. This article details the features which have been removed or changed in the ASP.NET Core SignalR.
+
+## Feature differences
+
+### Automatic reconnects
+
+Automatic reconnects are no longer supported. Previously, SignalR tried to reconnect to the server if the connection was dropped. Now, if the client is disconnected, the user must explicitly start a new connection to reconnect.
+
+### Protocol support
+
+ASP.NET Core SignalR supports JSON, as well as a new binary protocol based on [MessagePack](xref:signalr/messagepackhubprotocol). Additionally, custom protocols can be created.
+
+## Differences on the server
+
+The SignalR server-side libraries are included in the `Microsoft.AspNetCore.App` metapackage that is part of the **ASP.NET Core Web Application** template for both Razor and MVC projects.
+
+SignalR is an ASP.NET Core middleware, so it must be configured by calling `AddSignalR` in `Startup.ConfigureServices`.
```

Comment:
- SignalR --> ASP.NET Core SignalR
- Link to the AddSignalR API ref instead of code-fencing it. Here's the relative link to use: /dotnet/api/microsoft.extensions.dependencyinjection.signalrdependencyinjectionextensions.addsignalr#Microsoft_Extensions_DependencyInjection_SignalRDependencyInjectionExtensions_AddSignalR_Microsoft_Extensions_DependencyInjection_IServiceCollection_